### ISSUE TITLE
Images sent as <img> HTML element are not visible if alt text is not set

### ISSUE THREAD

**Issue by pkorba** [SchildiChat/schildichat-android-next#95]
**Describe the bug**
SchildiChat Next doesn't display inline images sent using \<img\> HTML tag if `alt` attribute is not set.

**To Reproduce**
1. Send HTML message with an image \<img\> (mxc url). 
Content of an example message:
```
{
  "content": {
    "body": "<img src=\"mxc://[redacted]\" height=\"100\" />",
    "format": "org.matrix.custom.html",
    "formatted_body": "<img src=\"mxc://[redacted]\" height=\"100\" />",
    "m.mentions": {},
    "msgtype": "m.text"
  },
  "origin_server_ts": 1767358887386,
  "sender": "@[redacted]",
  "type": "m.room.message",
  "unsigned": {
    "membership": "join",
    "age": 508,
    "transaction_id": "m1767358886914.3"
  },
  "event_id": "$[redacted]",
  "room_id": "![redacted]"
}
```
2. Image is not visible

If you send the same image message but add an `alt` attribute to \<img\> tag, image is being displayed properly. Example:
```
{
  "content": {
    "body": "<img src=\"mxc://[redacted]\" height=\"100\" alt=\"Example alt text\" />",
    "format": "org.matrix.custom.html",
    "formatted_body": "<img src=\"mxc://[redacted]\" height=\"100\" alt=\"Example alt text\" />",
    "m.mentions": {},
    "msgtype": "m.text"
  },
  "origin_server_ts": 1767358902512,
  "sender": "@[redacted]",
  "type": "m.room.message",
  "unsigned": {
    "membership": "join",
    "age": 1090,
    "transaction_id": "m1767358902285.4"
  },
  "event_id": "$[redacted]",
  "room_id": "![redacted]"
}
```
**Expected behavior**
Image should be visible even if no `alt` attribute is present within \<img\> element. Technically, `alt` attribute is mandatory, but in practice every other matrix client capable of displaying inline images and every browser displays images even if `alt` is not present.

**Screenshots**
First message contains image without `alt`. Second message contains image with `alt`.
<img width="468" height="448" alt="Image" src="https://github.com/user-attachments/assets/8fd2eda0-f1ae-45d8-984f-0e5f6c803a94" />

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S24 FE
 - OS: Android 16

**Additional context**
 - App version: 0.10.13 based on Element X 25.12.0
 - Store: Google Play
 - Homeserver: matrix.org
 - Related experimental/labs settings: No experimental settings
 - Logs have been submitted using the app's built-in bug report (/"rageshake") mechanism: no

**Upstream relevance**
 - Does this issue exist in Element X Android?
     - No, because as of now Element X Android cannot display \<img\> elements
